lib/views/login.js

```diff
@@ -40,69 +40,65 @@ function loginForm(req, res, locals) {
         next: req.query.next || '',
         forgotPasswordEnabled: config.get("mail").sendgridApiKey != ""
     }, locals)
-    
+
     res.send(pug.renderFile('templates/views/login.jade', locals))
 }
 
 function loginPost(req, res) {
     if (!req.body.email || !req.body.password) {
-	if (req.forceNoHTML || !req.accepts('text/html')) {
-	    res.status(401).type('text/plain').send('Please enter your e-mail address and password.')
-	    return
-	} else {
+        if (req.forceNoHTML || !req.accepts('text/html')) {
+            res.status(401).type('text/plain').send('Please enter your e-mail address and password.')
```

Comment:
Maybe "Please enter your username/e-mail address and password." instead, now that logon with a username is permitted :)?